frontend-react/src/content/resources/api-programmers-guide/ReportStreamApi.mdx

```diff
@@ -0,0 +1,66 @@
+import { CardGroup, Card, CardBody, CardHeader, ProcessList, ProcessListItem, ProcessListHeading  } from "@trussworks/react-uswds";
+
+# ReportStream API 
+
+<p className="usa-intro text-base">
+    By using the ReportStream Restful (REST) API, your data will automatically be filtered, transformed, batched, and forwarded to public health departments.
+</p>
+
+<CardGroup>
+    <Card gridLayout={{col: "fill"}}>
+        <CardHeader><span className="font-sans-lg">[Getting started](getting-started)</span></CardHeader>
+        <CardBody>Resources and support to set up your connection and format your data. Our onboarding guide will walk you through the steps needed to get started.</CardBody>
+    </Card>
+    <Card gridLayout={{col: "fill"}}>
+        <CardHeader><span className="font-sans-lg">[Documentation](documentation)</span></CardHeader>
+        <CardBody>Details on our data model and sample responses, inputs and outputs to assist you in our set up and use of the ReportStream API.</CardBody>
+    </Card>
+</CardGroup>
+
+<hr/>
+<br/>
```

Comment:
Instead of <br/> would <section></section> be a better option?

Reply:
If so, it would not be correct usage as the section should have a heading before it (ex: the content for Onboarding overview shortly after the first <br/> could be wrapped in a section). See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section